text/0000-network-update-loop.md

```diff
@@ -0,0 +1,720 @@
+- Feature Name: `network-update-loop`
+- Start Date: 2021-01-31
+- RFC PR: [Unity-Technologies/com.unity.multiplayer.rfcs#0000](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/0000)
+- Issue: [Unity-Technologies/com.unity.multiplayer#0000](https://github.com/Unity-Technologies/com.unity.multiplayer/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Often there is a need to update netcode systems like RPC queue, transport, and others outside the standard [`MonoBehaviour` event cycle](https://docs.unity3d.com/Manual/ExecutionOrder.html).
```

Comment:
Could you provide some example use cases? I'm not sure I understand why a user needs this.

Reply:
Network systems will be implemented on top of this infrastructure, some examples are RPC queue, transport IO and other potentials are also outlined in the "Future possibilities" at the bottom of the document.

Framework users may or may not utilize this skeleton — we could expect users to tie their game logic into some network systems running at the same stage.